packages/react/src/components/ImageCard/HotspotContent.jsx

```diff
@@ -84,7 +84,7 @@ const HotspotContent = ({
         <div className={`${iotPrefix}--hotspot-content-title-wrapper--editable`}>
           <TextInput
             className={`${iotPrefix}--hotspot-content-title-input`}
-            defaultValue={title}
+            defaultValue={typeof title === 'string' ? title : ''}
```

Comment:
This change was because title can be a react element, but that will throw an error passed as the default value. Another option here would be to use `title.toString()`